src/e2e/java/teammates/e2e/util/LNPSpecification.java

```diff
@@ -5,24 +5,18 @@
  */
 public class LNPSpecification {
 
-    /*
-        Note: Consider applying the builder pattern if this class gets more complicated.
-    */
-
+    // Maximum allowable threshold for the percentage of failed requests
+    // (0 to 100) to the test endpoint
```

Comment:
This should be proper Javadoc format instead of code comment.
```java
/**
 * Maximum allowable threshold for the ratio of failed request
 * (between 0 and 1) to the test endpoint.
 */
```
I modified the wordings a bit because the number is clearly a ratio, not percentage.